### DISCUSSION TITLE
Deploy Error: Process completed with exit code 1.

### DISCUSSION THREAD

**Discussion by yingxiong-gogreen** [alshedivat/al-folio#1634] [Q&A] [answered]
I am new to the GitHub page and encountered this deploy error message:
                    ------------------------------------------------
      Jekyll 4.3.2   Please append `--trace` to the `build` command 
                     for any additional information or backtrace. 
                    ------------------------------------------------
/opt/hostedtoolcache/Ruby/3.2.2/x64/lib/ruby/3.2.0/psych/parser.rb:62:in `_native_parse': (/home/runner/work/yingxiong-gogreen.github.io/yingxiong-gogreen.github.io/_config.yml): could not find expected ':' while scanning a simple key at line 16 column 1 (Psych::SyntaxError)

I have no idea why it goes like this accidentally, given my previous deployments are successful. I tried to use the initial folked _config.yml but still doesn't work. I really appreciate your help!

**Comment by riasat-sheikh** [comment, marked as answer]
If I am not wrong, maybe in some line (possibly `line 16`) inside the `_config.yml` file you have accidentally uncommented an available comment line or forgot to add a `:` after a YAML key. Please go through the config file once again.
